docs/EVG.md

```diff
@@ -180,6 +180,10 @@ Capitalized Terms are defined in the Baseline Requirements except where provided
 
 **Extended Validation Certificate:**  See EV Certificate.
 
+**Global Legal Entity Identifier Foundation:** Established by the Financial Stability Board in June 2014, the Global Legal Entity Identifier Foundation (GLEIF) is tasked to support the implementation and use of the Legal Entity Identifier (LEI). The foundation is backed and overseen by the LEI Regulatory Oversight Committee, representing public authorities from around the globe that have come together to jointly drive forward transparency within the global financial markets. GLEIF is a supra-national not-for-profit organization headquartered in Basel, Switzerland.
+
+**Global Legal Entity Identifier Index:** The Global Legal Entity Identifier (LEI) Index contains historical and current LEI records including related reference data in one authoritative, central repository. The reference data provides the information on a legal entity identifiable with an LEI. 
```

Comment:
This is also marketing copy, and can/should be deleted. It does not disambiguate in any meaningful sense, which is normally what these definitions should be used for.